.github/workflows/build.yml

```diff
@@ -276,6 +276,13 @@ jobs:
         run: |
           ../scripts/xcodebuild.sh macos/Example.xcworkspace test-without-building
         working-directory: example
+      - name: Build ARM
+        run: |
+          set -eo pipefail
+          yarn build:macos || yarn build:macos
+          pod install --project-directory=macos
```

Comment:
I don't think we need to do these again:
```suggestion
```

Reply:
Added, good catch.